frontend/src/app/components/containers/complaints/outcomes/hwcr-prevention-education.tsx

```diff
@@ -87,10 +87,10 @@ export const HWCRComplaintPrevention: FC = () => {
     if (complaintData) {
       const officer = getSelectedOfficer(assignableOfficers, personGuid, complaintData);
       setSelectedOfficer(officer);
-      dispatch(getPrevention(complaintData.id, officerList ?? undefined));
+      dispatch(getPrevention(complaintData.id));
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [complaintData]);
+  }, [complaintData, officers]);
```

Comment:
the same suggestion as the hwcr-complaint-assesment component, remove the officers object from the dependancy array and include the objects that should be there